### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by jefersonpmatos** [Elius94/react-photo-sphere-viewer#47]
### What happened?

The error continues to occur in Next.js 14.

![image](https://github.com/Elius94/react-photo-sphere-viewer/assets/92311057/b31090b4-1f5a-4ce6-9f1f-67f73e161544)

### What should have happened?

A bug happened!

### Library Version

[4.2.1-psv5.6.0](https://www.npmjs.com/package/react-photo-sphere-viewer/v/4.2.1-psv5.6.0)

### What operating system are you using?

Windows

### What browser are you using?

None

### Interest to fix the bug

- [ ] I would like to fix this bug!

**Comment by Elius94** [comment]
https://github.com/Elius94/react-photo-sphere-viewer?tab=readme-ov-file#usage

Hi! have you read this?

Please, be more specific if does not works.

**Comment by jefersonpmatos** [comment]
> https://github.com/Elius94/react-photo-sphere-viewer?tab=readme-ov-file#usage
> 
> Hi! have you read this?
> 
> Please, be more specific if does not works.

Yes, I followed the documentation for use in next, still the error persists. I would like to be more specific, but I don't have more details. The error already occurs with the simple example of use in next js. I'm using Next 14

***
![image](https://github.com/Elius94/react-photo-sphere-viewer/assets/92311057/01e87208-6c2b-4cba-ae43-ff13e0185460)

**Comment by mkubdev** [comment]
@jefersonpmatos Can you give more context ?

I tried to reproduce:

1. `npx create-next-app@14`
2. Select Javascript then all `yes`
3.
```bash
npm install -s @photo-sphere-viewer/core react-photo-sphere-viewer @photo-sphere-viewer/markers-plugin @photo-sphere-viewer/compass-plugin
```
4. Copy-pasted example from https://github.com/Elius94/react-photo-sphere-viewer?tab=readme-ov-file#usage on `app/page.js`
5. Everything work as expected..

**Comment by Elius94** [comment]
Everything is working also in my case! I close the issue for now :)
